workflow-templates/eslint.yml

```diff
@@ -0,0 +1,21 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Let's add main, master and develop branches

Reply:
okay i will add